doc/source/tutorials/emulator.rst

```diff
@@ -0,0 +1,153 @@
+.. _tutorials_emulator:
+
+How to use an emulator?
+=======================
+Basic Emulator Setups
+---------------------
+
+This tutorial will guide you through the process of setting up a one-qubit emulator using the QuTiP backend and showcase some of its features.
+
+First, we need to import the necessary modules. The main module we need is `create_oneQ_emulator` from `qibolab.oneQ_emulator`:
+
+.. code-block:: python
+
+    from qibolab.oneQ_emulator import create_oneQ_emulator
+
+Next, as with all platform create methods in Qibolab, we need the location of the folder containing the runcard that we would like to use. In this tutorial, we will use the demonstration runcard provided ("default_q0") with the emulator:
+
+.. code-block:: python
+    from qibolab.instruments.simulator import emulator_test
+
+    runcard_folder = f"{emulator_test.__path__[0]}/default_q0"
+
+Now, we can create the emulator using the `create_oneQ_emulator` function and the runcard folder:
+
+.. code-block:: python
+
+    emulator_platform = create_oneQ_emulator(runcard_folder)
+
+The emulator platform includes a pulse simulator, which we can access like this:
+
+.. code-block:: python
+
+    pulse_simulator = emulator_platform.instruments["pulse_simulator"]
+
+Finally, we can access the simulation backend of the pulse simulator.
+
+.. code-block:: python
+
+    simulation_backend = pulse_simulator.simulation_backend
```

Comment:
All of this should not be needed.

If the emulator exists, it should be loaded as any other Qibolab platform, just by instantiating the `QibolabBackend` object, or manually through `create_platform("my-emulator")`, as in here:
https://qibo.science/qibolab/stable/getting-started/experiment.html#run-the-experiment

Reply:
Fixed in 6f442c1ffbb9b110c9d3118cdeb67d354c19b42a